manifests/0000_03_operatorhub.cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: config.openshift.io/v1
```

Comment:
We want a filename that sorts after `0000_03_config-operator_01_operatorhub.crd.yaml` (which is what's currently vendored) and `0000_03_marketplace-operator_01_operatorhub.crd.yaml` (where openshift/api#1178 is taking the CRD).  Maybe `0000_03_marketplace-operator_02_operatorhub.cr.yaml`?